src/components/shared/thread/ActivityGraph.js

```diff
@@ -4,15 +4,10 @@
 // @flow
 
 import * as React from 'react';
-import { computeActivityGraphFills } from './ActivityGraphFills';
-import { timeCode } from 'firefox-profiler/utils/time-code';
+import { ActivityGraphCanvas } from './ActivityGraphCanvas';
 import classNames from 'classnames';
-import {
-  Tooltip,
-  MOUSE_OFFSET,
-} from 'firefox-profiler/components/tooltip/Tooltip';
-import SampleTooltipContents from 'firefox-profiler/components/shared/SampleTooltipContents';
-import { mapCategoryColorNameToStyles } from 'firefox-profiler/utils/colors';
+import { Tooltip, MOUSE_OFFSET } from '../../tooltip/Tooltip';
+import SampleTooltipContents from '../SampleTooltipContents';
```

Comment:
It looks like while working on this area, `firefox-profiler/` imports have been changed back to `../` imports. Would you mind changing them back to ``firefox-profiler/`?

Reply:
Done